collector/cluster-insight-controller.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Please add a comment (if possible):
this is the configuration file for deploying the cluster insight collector on the minion nodes.